core/lib/via_btc_client/src/types.rs

```diff
@@ -1,10 +1,160 @@
+use std::collections::VecDeque;
 use serde::{Deserialize, Serialize};
 use thiserror::Error;
+use bitcoin::script::PushBytesBuf;
+use bitcoin::taproot::Signature as TaprootSignature;
+use bitcoin::{Address as BitcoinAddress, Amount, TxIn, Txid};
+use zksync_basic_types::H256;
+use zksync_types::{Address as EVMAddress, L1BatchNumber};
 
 #[derive(Serialize, Deserialize)]
-pub enum BitcoinMessage {}
+pub enum BitcoinMessage {
+}
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub enum Vote {
+    Ok,
+    NotOk,
+}
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct CommonFields {
+    pub schnorr_signature: TaprootSignature,
+    pub encoded_public_key: PushBytesBuf,
+    pub via_inscription_protocol_identifier: String,
+}
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct L1BatchDAReferenceInput {
```

Comment:
this is not correct 

we need blob id from da_client library 
or simpler call it `blob_id:String`